core/src/test/java/global/maplink/domain/MaplinkPointsTest.java

```diff
@@ -49,6 +50,31 @@ class MaplinkPointsTest {
             }
     };
 
+    @Test
+    void shouldBeFillableFromDoubleArray() {
+        MaplinkPoints points = MaplinkPoints.from(
+                -23.56649,
+                -46.6538,
+                -23.5658,
+                -46.65313,
+                -23.56486,
+                -46.65221,
+                -23.56486,
+                -46.65221,
+                -23.56161,
+                -46.65613,
+                -23.56162,
+                -46.65615
+        );
+
+        assertThatMatchWithSample(points);
+    }
+
+    @Test
+    void shouldFailWhenInvalidDoubleArrayIsPassed() {
+        assertThatThrownBy(() -> MaplinkPoints.from(-23.56649, -46.6538, -23.5658)).isInstanceOf(IllegalArgumentException.class);
```

Comment:
Não seria bom ter certeza de que a mensagem bate com "MaplinkPoints.from requires coordinates in pairs"?

Reply:
pensei em não tornar o teste tão rigido, já que a msg pode mudar, importante é falhar quando o valor não for par, já que não é um conjunto válido de coordenadas

Reply:
é que assim não garante que está pegando o problema do número ímpar (par quebrado de coordenada), pois se o cara mandou um argumento inválido tipo com vírgula em vez de ponto decimal ou ainda com os sufixos W, S, E, N poderia furar no mesmo caso de IllegalArgumentException né? Ou não?

Reply:
na verdade aqui só aceita double mesmo
erro de separador de decimal ou caracter nem passa por essa validação, isso é com o jackson